src/sections/Company/Brand/Brand-components/meshmap.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styled from "styled-components";
+import { SRLWrapper } from "simple-react-lightbox";
+import { Row, Col } from "../../../../reusecore/Layout";
+import Button from "../../../../reusecore/Button";
+import ColorBox from "../../../../components/ColorBox";
+import { FiDownloadCloud } from "@react-icons/all-files/fi/FiDownloadCloud";
+import MeshMapIcon from "../../../../assets/images/meshmap/meshmap-color-icon.png";
+import MeshMap from "../../../../assets/images/meshmap/meshmap-color-stacked.png";
+import MeshMapTitle from "../../../../assets/images/meshmap/meshmap-color-text.png";
```

Comment:
Only use `SVG` versions of the image.
Though, keep the png files under the assets directory (what you have done already)

Reply:
Give me few mins, will be doing it now.